common/lib/xmodule/xmodule/course_module.py

```diff
@@ -444,7 +445,7 @@ class CourseFields(object):
                         "short_description": _("Explore the Studio Help Forum"),
                         "long_description": _("Access the Studio Help forum from the menu that appears when you click your user name in the top right corner of Studio."),
                         "is_checked": False,
-                        "action_url": "http://help.edge.edx.org/",
+                        "action_url": settings.TENDER_URL + "/",
```

Comment:
This line makes this PR less straightforward than I thought, because common doesn't have access to django settings